tests/samplers_tests/test_samplers.py

```diff
@@ -12,6 +12,7 @@
 
 if optuna.types.TYPE_CHECKING:
     import typing  # NOQA
+    from typing import Any  # NOQA
```

Comment:
Please remove this line because `Any` is not used in this file.

Reply:
Thank you...! Fixed: https://github.com/pfnet/optuna/pull/441/commits/54e675dd58f9898d7b4de463f55f93508d4db7e6